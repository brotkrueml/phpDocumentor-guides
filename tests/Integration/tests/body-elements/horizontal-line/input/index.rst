
Some Document
=============

Testing separator

-------

Hey!
