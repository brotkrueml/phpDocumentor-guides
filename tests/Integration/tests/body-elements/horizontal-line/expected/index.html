<!DOCTYPE html>
<html lang="en">
<head>
    <title>Some Document</title>

</head>
<body>
<div class="section" id="some-document">
    <h1>Some Document</h1>

    <p>Testing separator</p>
    <hr />

    <p>Hey!</p>
</div>

</body>
</html>
